test/e2e_node/util.go

```diff
@@ -365,3 +367,26 @@ func runCommand(cmd ...string) (string, error) {
 	}
 	return string(output), nil
 }
+
+// getCRIClient connects CRI and returns CRI runtime service clients and image service client.
+func getCRIClient() (internalapi.RuntimeService, internalapi.ImageManagerService, error) {
+	// connection timeout for CRI service connection
+	const connectionTimeout = 15 * time.Minute
```

Comment:
Why setting such a high timeout? kubelet's default timeout is 2 minutes, right?

Reply:
Copied from `image_list.go` I can change it to 2 minutes.

Reply:
Done.